server/EventsFacade/Utilities/RegisterEventsModule.cs

```diff
@@ -0,0 +1,22 @@
+﻿using EventStore.Client;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace EventsFacade.Utilities
+{
+    public static class RegisterEventsModule
+    {
+        private const string EventStoreConnectionString = "esdb://eventstore.db:2113?tls=false";
```

Comment:
Can it be moved to some configuration file?

Reply:
Sure 👍🏽